### ISSUE TITLE
Bluespace bodybags runtime on creation

### ISSUE THREAD

**Issue by 81Denton** [tgstation/tgstation#42105]
```
[14:56:12] Runtime in _component.dm,87: Cannot read "atom_can_reach".gc_destroyed
  proc name: RegisterSignal (/datum/proc/RegisterSignal)
  src: the bluespace body bag (/obj/item/bodybag/bluespace)
  src.loc: the floor (148,220,9) (/turf/open/floor/plasteel/grimy)
  call stack:
  the bluespace body bag (/obj/item/bodybag/bluespace): RegisterSignal("atom_can_reach", /datum/callback (/datum/callback), null, 0)
  the bluespace body bag (/obj/item/bodybag/bluespace): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the bluespace body bag (/obj/item/bodybag/bluespace), /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms(null)
  Atoms (/datum/controller/subsystem/atoms): Initialize(501640)
  Master (/datum/controller/master): Initialize(10, 0, 1)
```

**Comment by SpaceManiac** [comment]
Fixed by #42088. No round ID provided so I'm assuming the server is outdated.

**Comment by 81Denton** [comment]
Dang, I must have run an outdated revision.
